ros/src/computing/perception/detection/trafficlight_recognizer/launch/traffic_light_recognizer_tensorflow.launch

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- Arguments -->
+  <arg name="camera_id" default="camera"/>
+  <arg name="nn_model_path" />
+  <arg name="image_transport" default="compressed" />
+  <arg name="image_raw_topic" default="$(arg camera_id)/image_raw "/>
+
+  <!-- ROI extractor -->
+  <node pkg="trafficlight_recognizer" type="roi_pub" name="roi_pub">
+      <param name="image_transport" value="$(arg image_transport)" />
+      <param name="image_raw_topic" value="$(arg image_raw_topic)" />
+  </node>
+  
+  <!-- Classifier -->
+  <node pkg="trafficlight_recognizer" type="region_tlr_tensorflow.py" name="region_tlr_tensorflow">
+      <param name="nn_model_path" value="$(arg nn_model_path)" />
+      <!-- param name="swap_pole_lane_id" value="false"/ -->
+  </node>
+  
+  <!-- ROI output -->
+  <node pkg="image_view" type="image_view" name="Old" args="image:=$(arg image_raw_topic) _image_transport:=$(arg image_transport)"/>
+  <node pkg="image_view" type="image_view" name="New" args="image:=/tlr_roi_image"/>
```

Comment:
Also remove these. People can run their own image_view and won't want these popping up unexpectedly.